Source/WebKit/UIProcess/ios/WKContentViewInteraction.h

```diff
@@ -862,6 +871,14 @@ FOR_EACH_PRIVATE_WKCONTENTVIEW_ACTION(DECLARE_WKCONTENTVIEW_ACTION_FOR_WEB_VIEW)
 - (void)_didChangeDragCaretRect:(CGRect)previousRect currentRect:(CGRect)rect;
 #endif
 
+-(void)panGestureDidBeginAtPoint:(CGPoint)inputPoint;
+-(void)panGestureDidUpdateWithPoint:(CGPoint)inputPoint;
+-(void)panGestureDidEnd;
```

Comment:
Are these only called from within `WKContentViewInteraction.mm`? If so, they don't need to be declared in this header.